docs/SimpleFormIterator.md

```diff
@@ -150,18 +150,20 @@ const PostEdit = () => (
 **Note**: `<SimpleFormIterator>` only accepts `Input` components as children. If you want to use some `Fields` instead, you have to use a `<FormDataConsumer>` to get the correct source, as follows:
```

Comment:
This is actually incorrect, it accepts fields, but without a label, once you add the Labeled component you need the `FormDataConsumer`

Reply:
Well, it depends: you can use Label+Fields directly to display the `record` value, but if you want to render a reactive value based on the form inputs, you have to use the formDataConsumer. 

I think this should be clarified in the doc.

Reply:
Well, actually, this is how it goes inside a `SimpleFormIterator`:

```jsx
<ArrayInput source="authors">
    <SimpleFormIterator>
        <TextField source="role" />        // => Value is shown , no label
        <Labeled label="Role">              // => Label is shown , no value
            <TextField source="role" />
        </Labeled>
        <FormDataConsumer>               // => Label + value are shown
            {({ getSource }) => {
                return (
                    <Labeled label="Role">
                        <TextField source={getSource('role')} />
                    </Labeled>
                );
            }}
        </FormDataConsumer>
        <FormDataConsumer>                   // => Label is shown, no value
            {({ getSource, scopedFormData }) => {
                return (
                    <Labeled label="Role">
                        <TextField
                            source={getSource('role')}
                            record={scopedFormData}
                        />
                    </Labeled>
                );
            }}
        </FormDataConsumer>
    </SimpleFormIterator>
</ArrayInput>
```